### PR TITLE
Exclude the pdb in the mcc submission

### PR DESCRIPTION
**Pull request by jashook**

### CODE REVIEW

File: src/coreclr/tests/testgrouping.proj [hunk 1]
```diff
@@ -10,7 +10,7 @@
       <XUnitWrapperDll>$(BinDir)JIT\jit64\JIT.jit64.XUnitWrapper.dll</XUnitWrapperDll>
     </TestGrouping>
 
-    <TestGrouping Condition="'$(TargetOS)' == 'Windows_NT'" Include="$(BinDir)JIT\jit64\mcc\**">
+    <TestGrouping Condition="'$(TargetOS)' == 'Windows_NT'" Include="$(BinDir)JIT\jit64\mcc\**"  Exclude=”$(BinDir)JIT\jit64\mcc\**\*.pdb”>
```
Comment (echesakovMSFT) [inline src/coreclr/tests/testgrouping.proj:13]: Oops, there are Unicode quotes - need to replace them with "plain" ones.

Comment (jashook) [reply src/coreclr/tests/testgrouping.proj:13]: Beautiful thanks

**Comment by echesakovMSFT** [comment]
@jashook Can you please also trigger pri1 tests? I believe some of the mcc tests are pri1.

**Comment by jashook** [comment]
Initial data:

```
[With PDB] Pri0 setup time: 0:05:42.956000, Pri0 run time: 0:00:17.329000. Setup percentage 95%
[Without PDB] Pri0 setup time: 0:00:29.732000, Pri0 run time: 0:00:14.655000. Setup percentage 66%
[With PDB] pri1 setup time: 0:05:32.667000, pri1 run time: 0:00:27.862000. Setup percentage 92%
```

**Comment by jashook** [comment]
Rerunning outerloop as it needed a rebase

**Comment by jashook** [comment]
@echesakovMSFT I think there is enough information now. I have an outerloop run going again but the setup time looks the same as pri0, so I expect the exact same numbers.

**Comment by jashook** [comment]
The information validates the existing guidance that smaller workitems give large setup time reductions.

**Comment by jashook** [comment]
Note that removing the pdbs dropped the zip file size from 701mb to 90mb. I can imagine that of the ~5 minutes to setup the payload 4+ minutes is downloading

**Pull request closed without merging**